test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelperSampleTest.cs

```diff
@@ -18,44 +22,110 @@ public TagHelperSampleTest(MvcTestFixture<TagHelperSample.Web.Startup> fixture)
 
         public HttpClient Client { get; }
 
-        public static TheoryData<string> PathData
+        [Fact]
```

Comment:
This isn't a `[Fact]` it's a couple of `[Theory]`s, one that expects `HttpStatusCode.OK` and another `(HttpStatusCode.Redirect`. Should assert the redirect URL in the second `[Theory]`.